coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -6714,22 +6714,33 @@ def test_atan2_broadcast(self, compute_unit, backend, frontend, rank):
 
 class TestTriu(TorchBaseTest):
     @pytest.mark.parametrize(
-        "compute_unit, backend, frontend, shape, diagonal",
+        "compute_unit, backend, frontend, shape, diagonal, dtype",
         itertools.product(
             compute_units,
             backends,
             frontends,
             [(5, 5), (3, 4), (5, 1)],
             [None, -1, 0, 2],
+            [torch.float16, torch.int32, torch.bool],
         ),
     )
-    def test_triu(self, compute_unit, backend, frontend, shape, diagonal):
+    def test_triu(self, compute_unit, backend, frontend, shape, diagonal, dtype):
         params_dict = {}
         if diagonal is not None:
             params_dict["diagonal"] = diagonal
         model = ModuleWrapper(torch.triu, params_dict)
+        input_data = torch.randn(shape)
```

Comment:
Doesn't really make sense to populate this only to have it usually override. I'd put this line in an else-block.